Feature/form page

anelbas opened the pull request:
Add initial form page (@media queries still to be made to the form. Modal completed)
Update navbar code.

anelbas committed:
Create adoption form page and update navbar

anelbas committed:
Add adoption form page and updage navbar

AndrewBB-BBD reviewed (commented):
This is so cool! Well done, Anel! :D 

My comments and questions:
- Text input fields: the text you type in is lighter than the text that is displayed when the field is empty. Could we switch that around so that what you type in is darker than the placeholder text? The colours are up to you.
- Hover effects: Could you please add hover effects to the buttons, like the "What is required of me" text? We could beef this up by changing both the cursor and the shade of the button when you hover on it. I'm not sure whether to make the button lighter or darker, or whether to change border colour, but an effect would be cool and we could use it throughout the app. It's also common button behaviour.
- When I load the page, the modal is always open. Should it be like this or am I doing something wrong?
- You said you'd do the media queries tomorrow. Perhaps we can hop on a call and chat about the modal?

AndrewBB-BBD commented on Client-Side/adoptForm/adoptForm.html:
```diff
@@ -0,0 +1,54 @@
+    <main class="mainBody">
+        <form action="" class="adoptionForm">
+            <header>Take the first step to take your<br>fluffy friend home!</header>
+            <a class="sliderLink" href="">What is required of me?</a><br><br>
```
I think < a > is more for a hyperlink. Perhaps use a button with "text-decoration: underline" and "background: none"? 
I'm not so sure of this so please fight me if you want

anelbas replied:
Made the change

AndrewBB-BBD commented on Client-Side/adoptForm/adoptForm.html:
```diff
@@ -0,0 +1,54 @@
+            <header>Take the first step to take your<br>fluffy friend home!</header>
+            <a class="sliderLink" href="">What is required of me?</a><br><br>
+            <label for="inputFirstName">FIRST NAME *</label><br>
+            <input type="text" name="inputFirstName" placeholder="Enter your first name"><br><br>
```
< br > tag  shouldn't be used to space things outside of text. Rather use "margin-bottom: "

AndrewBB-BBD commented on Client-Side/adoptForm/adoptForm.html:
```diff
@@ -0,0 +1,54 @@
+            <button class="submitButton">Please contact me</button>
+        </form>
+
+        <aside class="modalContainer">
```
Use < dialog > tag for the modal

anelbas replied:
Made the change

anelbas commented:
Corrected the input text color to a darker gray and added hover effects (I used the same one on the yellow button as Maddie used on the home page. The color isn't the best though so it might be changed later). The modal is hidden on page load now.

We can chat about the media queries tomorrow. It will be in the afternoon though because I am out the whole morning.

Thanks or your pointers, they were super insightful & helpful. Appreciate it :)

anelbas committed:
Updates to adoptForm page as instructed by Andrew

AndrewBB-BBD committed:
feat: media queries

anelbas committed:
Change input tag sizing

anelbas committed:
Update input types
